### ISSUE TITLE
Is there setDate() method?

### ISSUE THREAD

**Issue by AdamReznicek** [xdan/datetimepicker#720]
Is it possible to set date dynamically via javascript after calendar init?

**Comment by psaikido** [comment]
Yes, I too need to set a date for testing purposes. Mocha's setValue() won't set the date and I'm struggling to find a method of doing it.

**Comment by quim831** [comment]
Hello
Did you find a way to set a Date?
Thanks
